src/jmcomic/jm_downloader.py

```
@@ -226,35 +257,50 @@ def before_photo(self, photo: JmPhotoDetail):
+        try:
+            self.raise_if_cancelled()
+            super().after_image(image, img_save_path)
+            self.option.call_all_plugin(
+                'after_image',
+                image=image,
+                downloader=self,
+            )
+        except DownloadCancelledException as error:
+            cancellation_error = error
+
+        # 正常完成或取消时登记；普通插件异常直接向外传播，不进入成功清单。
```

Ruff (0.16.3)

[warning] Comment contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF003)